eng/pipelines/templates/stages/archetype-sdk-tool-repo-sync.yml

```diff
@@ -165,20 +165,6 @@ stages:
                         -AuthToken "$(azuresdk-github-pat)"
                       pwsh: true
 
-                  - task: PowerShell@2
```

Comment:
At some point if we want to use the real github automerge feature we can replace this with https://github.com/Azure/azure-sdk/blob/main/eng/pipelines/version-updater.yml#L91-L97